quicktill/printer.py

```diff
@@ -25,6 +27,9 @@ def print_receipt(printer, transid):
     if not trans.lines:
         return
     with printer as d:
+        if tillconfig.publogo:
+            image = base64.b64decode(f"{tillconfig.publogo}")
```

Comment:
This should be:
```
if tillconfig.publogo():
    image = base64.b64decode(tillconfig.publogo())
```
(`tillconfig.publogo` will always test True)

Reply:
Ack, good catch!